src/Background.cpp

```diff
@@ -1,9 +1,9 @@
 #include "Background.h"
+#include "ResourceManager.h"
 
 Background::Background()
 {
-	texture.loadFromFile("Res/quackMeadowBG.png");
-
+	texture = ResourceManager::Get().GetTexture("Meadow");
```

Comment:
the whole point of having a resource manager is not having to deal with resources anywhere else :)

classes like Background, shouldn't have textures, soundBuffers, fonts, etc as their members. Only classes that actually use them.

So, delete the texture, leave only sprite, and then you just 
`bg.setTexture(ResourceManager::Get().GetTexture("Meadow"));`